apps/mcp/graph-ui/src/styles.css

```
@@ -47,3 +47,426 @@ body {
 }
+
+/* Workspace-first product surface */
+.workspace-shell {
+  --workspace-bg: #f5f4ef;
+  --workspace-panel: #fffefa;
+  --workspace-ink: #17211d;
+  --workspace-muted: #6d756f;
+  --workspace-line: #dedfd8;
+  --workspace-green: #276749;
+  --workspace-green-soft: #e7f2ea;
+  min-height: 100vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)